### ISSUE TITLE
Prevent failed login

### ISSUE THREAD

**Issue by imclint21** [3proxy/3proxy#1103]
Hi,

There is any possibilities to ban people that try too much time to connect?

Regards

**Comment by z3APA3A** [comment]
Use fail2ban, you should ban IPs, not people.

**Comment by imclint21** [comment]
Any resources about this? Create some regex based on error code right?

**Comment by z3APA3A** [comment]
Yes, count errors 5 and 6 (or 7,8 depending on stored password form) https://3proxy.ru/howtoe.asp#ERRORS

fail2ban supports 3proxy out-of box, I'm not sure it uses default logformat thought.

**Comment by imclint21** [comment]
What do you mean by "supports 3proxy out-of box"?

**Comment by z3APA3A** [comment]
https://github.com/fail2ban/fail2ban/blob/master/config/filter.d/3proxy.conf

**Comment by imclint21** [comment]
ah wonderful 👌🏼
